specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2016-09-01-preview/vmwarevirtustream.json

```diff
@@ -0,0 +1,1438 @@
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.VMwareVirtustream/privateClouds/{privateCloudName}/addAuthorization/{authorizationName}": {
+      "post": {
+        "operationId": "PrivateClouds_AddAuthorization",
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "PrivateClouds"
+        ],
+        "summary": "adds a number of hosts to a cluster",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "name": "privateCloudName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the privateCloud."
+          },
+          {
+            "name": "authorizationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the ER authorization."
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/PrivateCloudResponse"
+            }
+          },
+          "202": {
+            "description": "accepted"
+          }
+        },
+        "x-ms-examples": {
+          "PrivateClouds_AddAuthorization": {
+            "$ref": "./examples/PrivateClouds_AddAuthorization.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.VMwareVirtustream/privateClouds/{privateCloudName}/deleteAuthorization/{authorizationName}": {
+      "post": {
+        "operationId": "PrivateClouds_DeleteAuthorization",
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "PrivateClouds"
+        ],
+        "summary": "adds a number of hosts to a cluster",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "name": "privateCloudName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the privateCloud."
+          },
+          {
+            "name": "authorizationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the ER authorization."
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
```

Comment:
Should this have a 204?

Reply:
No, only the resources are responding with 204. This endpoint does not. Should it. I could put in a request to change it for the next API version.